package_manifest.yaml

```diff
@@ -12,6 +12,9 @@ packages:
   vars:
     rpmlint_rc_file: "{{ inventory_dir }}/.rpmlintrc"
     build_package_use_koji_build: true
+    macros:
```

Comment:
this makes the `macros` var *always* available and I am not 100% you had intended this? (if you did, cool, just wanted to explicitly point it out)